### PR TITLE
Bump version to v0.3.0

### PR DESCRIPTION
**Pull request by feedmeapples**
## Checklist

1. Closes 

2. How was this tested:

tests

3. Any docs updates needed?

### CODE REVIEW

**Review by bergundy — APPROVED** [review]
When we create the release will the archive name be `temporal-cli...`?

**Comment by feedmeapples** [comment]
> 

not yet, sending a PR renaming the archive name. Keeping the current PR open until merging the new archive name

File: headers/headers.go [hunk 1]
```diff
@@ -17,7 +17,7 @@ const (
 const (
 	ClientNameCLI = "temporal-cli"
 
-	CLIVersion = "0.2.0" // TODO switch version to "1.18.1.0" once server accepts new format
+	CLIVersion = "0.3.0" // TODO switch version to "1.18.1.0" once server accepts new format
```
Comment (feedmeapples) [inline headers/headers.go:20]: Since goreleaser doesn't allow this 

```suggestion
	CLIVersion = "0.3.0"
```

Comment (feedmeapples) [inline headers/headers.go:20]: Since goreleaser doesn't allow such format

```suggestion
	CLIVersion = "0.3.0"
```

**Pull request merged**